packages/perseus/src/perseus-types.ts

```diff
@@ -731,6 +736,19 @@ export type LockedPolygonType = {
     strokeStyle: "solid" | "dashed";
 };
 
+export type LockedFunctionType = {
+    type: "function";
+    color: LockedFigureColor;
+    strokeStyle: "solid" | "dashed";
+    equation: string;
+    equationParsed?: {
```

Comment:
Could you add comments here about the usage for / difference between `equation` and `equationParsed`?

Reply:
Added.